### DISCUSSION TITLE
Changing driver imperfection from within simulation

### DISCUSSION THREAD

**Discussion by pv2-0** [eclipse/mosaic#90] [Q&A] [answered]
Hi, I am trying to change the driver imperfection value of some vehicles through an application. Is this possible? I know that I can access the default sigma value from the Vehicle Type class but am I able to edit it?

Thanks

**Comment by kschrab** [comment, marked as answer]
You can do so by calling

```java
getOs().requestVehicleParametersUpdate().changeImperfection(0.4).apply();
```
